scripts/index.js

```diff
@@ -1,51 +1,188 @@
+    image: './assets/images/deliveroo.svg',
+    source: 'https://github.com/sireclifford/sire-portfolio',
+  },
+
+];
+
+const loadRecentWork = () => {
+  // define media queries
+  const mobileMediaQuery = window.matchMedia('(max-width: 600px)');
+
+  const work = document.querySelector('.recent-work-list');
+
+  const desktopClose = document.getElementsByClassName('desktop-close-popup')[0];
 
-const singleProjectModal = document.getElementById('single-project-modal');
-const singleProjectBtn = document.querySelector('.single-project-btn');
-singleProjectBtn.addEventListener('click', () => {
-  singleProjectModal.style.display = 'block';
-});
+  const projects = portfolioData.map((project) => `<div class="work-card">
+      <div class="card-header">
+        <img src="${project.image}" alt="${project.title}"/>
+      </div>
+      <div class="card-body">
+        <h1>${project.title}</h1>
+        <ul>
+        ${project.skills.map((item) => `<li>${item}</li>`).join('')}
+        </ul>
+        <a id="${project.id}" href="#" class="btn-success btn-three ${mobileMediaQuery.matches ? 'single-project-btn' : 'desktp-view-project-btn'}"
+          >See Project</a
+          single-project-btns
+        >
+      </div>
+    </div>`).join('');
+
+  const range = document.createRange();
+
+  range.selectNode(work);
+  const fragment = range.createContextualFragment(projects);
+  work.appendChild(fragment);
+
+  // Place scripts here
+  const singleProjectModal = document.querySelector('#single-project-modal');
+  const singleProjectBtn = document.querySelectorAll('.single-project-btn');
+  singleProjectBtn.forEach((btn) => {
+    btn.addEventListener('click', (event) => {
+      const foundProject = portfolioData.filter((x) => String(x.id) === event.target.id);
+      document.getElementsByClassName('single-project-pic')[0].src = foundProject[0].image;
+      document.getElementsByClassName('single-project-pic')[0].alt = foundProject[0].title;
+      document.getElementsByClassName('single-project-title')[0].innerHTML = foundProject[0].title;
+      document.getElementsByClassName('single-project-skills')[0].innerHTML = foundProject[0].skills.map((item) => `<li>${item}</li>`).join('');
+      document.getElementsByClassName('single-project-desc')[0].innerHTML = foundProject[0].description;
+      document.getElementsByClassName('project-source')[0].href = foundProject[0].source;
+      singleProjectModal.style.display = 'block';
+      const close = document.getElementsByClassName('close-popup')[0];
+      close.onclick = () => {
+        singleProjectModal.style.display = 'none';
+        range.detach();
+      };
+    }, true);
+  });
+
+  const desktopModal = document.querySelector('.desktop-modal');
```

Comment:
- [ ] Great work you did on your modal, kindly check I notice that in mobile view the modal popup doesn't work, as a requirement the popup is supposed to be responsive and fit mobile and desktop sizes properly.

Reply:
@danielufeli the mobile popups work as well. Kindly review how you checked it or point me to how it's not working.

Reply:
When I click the button it takes me to the top of the page. Without showing the popup.

Reply:
@danielufeli attached are screenshots of it working on my side. 
When I click on the `see project` button. And I also have my partner @TracK92 confirming it.
I will be glad if you can show me which button gives you that effect.

<img width="334" alt="Screenshot 2022-06-17 at 12 05 31 PM" src="https://user-images.githubusercontent.com/82734569/174295111-f347aadc-a06b-4cab-81e1-2cc39e8cdfdb.png">
<img width="334" alt="Screenshot 2022-06-17 at 12 05 28 PM" src="https://user-images.githubusercontent.com/82734569/174295185-6a529caf-cd49-47dd-b2ee-8920fa7d1fb2.png">
<img width="327" alt="Screenshot 2022-06-17 at 12 05 20 PM" src="https://user-images.githubusercontent.com/82734569/174295205-d5bfb6cd-9af8-488f-b731-7d8f10187083.png">
<img width="338" alt="Screenshot 2022-06-17 at 12 05 14 PM" src="https://user-images.githubusercontent.com/82734569/174295217-871ae9c3-55ac-4817-84f3-5724e2ea4b4f.png">

Reply:
@sireclifford is still the same on my end when I scroll to the bottom of the page I see this:

![--mobile--](https://user-images.githubusercontent.com/48371156/174303390-6ce678c7-4f7a-4996-81a9-f8b843617c08.png)

Reply:
@sireclifford after doing npm install it works fine now.